library/src/main/java/com/novoda/downloadmanager/NotificationCustomizer.java

```diff
@@ -7,6 +7,13 @@
 
     NotificationDisplayState notificationDisplayState(T payload);
```

Comment:
Specify that this should return the display state (with a link to the class) for the given payload, so that the user can decide if to show/hide the notification.
Also, missing javadoc for that class